universal-application-tool-0.0.1/app/views/components/QuestionBank.java

```diff
@@ -160,14 +159,13 @@ private ContainerTag renderQuestionDefinition(QuestionDefinition definition) {
    * <p>Questions that are filtered out:
    *
    * <ul>
-   *   <li>If there is at least one question in the current block, all enumerator questions
+   *   <li>If there is at least one question in the current block, all single-block questions
    *   <li>If this is a repeated block only show the appropriate repeated questions
    *   <li>Questions already in the program
```

Comment:
(blocking) Is this missing a case?

`If there is single block question, all questions are filtered.`

Reply:
Done.